### ISSUE TITLE
Utterly Broken UI on MacOS Safari

### ISSUE THREAD

**Issue by BenSapiens** [go-gitea/gitea#5855]
- Gitea version (or commit ref): v1.7
- Git version: N/A
- Operating system: MacOS Mojave 10.14.2
- Database: N/A
- Can you reproduce the bug at https://try.gitea.io: No.
- Log gist: N/A

## Description w/ screenshots

When I browse my Gitea installation with Chrome or Firefox, everything works beautifully. When I attempt to browse with Safari, the UI breaks completely, viz. https://i.imgur.com/S4ty18G.png.

I had a poke around with the Web Inspector. This is what's shown in the console: https://i.imgur.com/NdMNt5s.png

And in the network panel it appears that both `semantic.min.js` and `semantic.min.css` are failing to load entirely: https://i.imgur.com/SdgkT7l.png

I double-checked my installation directory, a freshly fetched repository checked out to branch `release/v1.7`, and both files are right where they're supposed to be.

Furthermore, when I visit the demo at `try.gitea.io` it works fine.

I have no explanation for this. Perhaps I'm missing something basic.

I would provide logs but I can't seem to find any relevant ones. Nothing in `/var/lib/gitea/log` seems to correspond to anything like this.

**Comment by 0x5c** [comment]
`MacOS Mojave 10.14.2`
Is that the host, the client, or both?

**Comment by lunny** [comment]
Could you find the error when loading `semantic.min.js` and `semantic.min.css`?

**Comment by BenSapiens** [comment]
On the server? No, if it exists, I was unable to locate it. With a recursive grep in `/var/lib/gitea/log`, the search "semantic" yields nothing. I doubt there's a useful error anywhere, but I'll provide whichever logs you want if you want them.

**Comment by BenSapiens** [comment]
Or if you have in mind some other way I can help.

**Comment by lunny** [comment]
Which version of your safari? Cannot reproduce on my macOS Mojave 10.14.3 and Safari 12.0.3 (14606.4.5)

**Comment by BenSapiens** [comment]
I have those exact version numbers.

I just checked with a fresh user, and it's still broken.

I also tried with Opera, works fine there. In addition, I spoofed my Safari user agent with a Firefox extension. Still fine.

Maybe it's on the server? Would a compile log help? The server's running CentOS 7, as I said before, so the packages are horrendously out of date.

I don't know, I'll help if I can, but whether you want to spent any significant time on this is up to you.

Or, since we have the exact same OS and browser, I could PM you the URL of the server in question, just to see if it's broken for you as well.
